### PR TITLE
Changelog 22

### PR DESCRIPTION
**Pull request by deltaepsilon**
image

### CODE REVIEW

File: docs-content/general/changelog/changelog-22.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+[@Vadman97](https://github.com/Vadman97) solved a long-standing issue with WebGL double buffering. The WebGL memory would render a transparent image even though the GPU was rending the image correctly. This caused missing images in Session Replay.
+
+Take full control over your Canvas snapshots with the [manual snapshotting docs](https://www.highlight.io/docs/getting-started/client-sdk/replay-configuration/canvas#manual-snapshotting)
+
+![Render to Highlight](/images/changelog/22/canvas-snapshot.jpg)
+
+
+## Set "Service Name" and "Service Version" with Python SDK
+
+We've added `service_name` and `service_version` parameters to the [Python SDK](https://www.highlight.io/docs/sdk/python#servicename)
```
Comment (et) [inline docs-content/general/changelog/changelog-22.md:32]: Add something for Go as well: https://www.highlight.io/docs/sdk/go#highlightStart

File: docs-content/general/changelog/changelog-22.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+These params make logs much more searchable.
+
+![Python Service Name](/images/changelog/22/service-name.png)
+
+## 🚨Community Contribution 🚨 
+
+#### Repaired scrolling in metadata
+
+Shout out to to Kalkidan Betre—[@kalibetre](https://github.com/kalibetre) on GitHub—for his open source contribution. He noticed that our Metadata window on the Session Replay screen wasn't scrolling correctly, and he sent in a pull request to fix it.
```
Comment (et) [inline docs-content/general/changelog/changelog-22.md:42]: `for his open` -> `for their open`

Comment (et) [reply docs-content/general/changelog/changelog-22.md:42]: `He noticed that our` -> `They noticed`

Comment (et) [reply docs-content/general/changelog/changelog-22.md:42]: `and he sent` -> `and they sent`

File: docs-content/general/changelog/changelog-22.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Thanks Kalkidan!
+
+![Metadata scroll](/images/changelog/22/metadata-scroll.gif)
+
+
+
+
+
```
Comment (et) [inline docs-content/general/changelog/changelog-22.md:51]: Minor nit - do we need all these blank lines?

**Review by et — APPROVED** [review]
Small bit of feedback but nothing blocking.

File: docs-content/general/changelog/changelog-22.md (hunk 1 above)
Comment (deltaepsilon) [reply docs-content/general/changelog/changelog-22.md]: Done!

**Pull request merged**